ingress-nginx.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Ingress NGINX Controller
```

Comment:
I've only heard of it referred to it as "Nginx Ingress Controller"
```suggestion
title: NGINX Ingress Controller
```

Reply:
done, however have issue where the netlify build process fails due to  having a 1.x update after 2.x has started.

Have logged as https://github.com/endoflife-date/endoflife.date/issues/6206